hnn_core/viz.py

```diff
@@ -488,19 +499,64 @@ def plot_spikes_hist(cell_response, trial_idx=None, ax=None, spike_types=None,
         spike_type_times[spike_label].extend(
             spike_times[spike_types_mask[spike_type]])
 
+    if invert_spike_types is None:
+        invert_spike_types = list()
+    else:
+        if isinstance(invert_spike_types, dict):
```

Comment:
```suggestion
        if not isinstance(invert_spike_types, (str, list)):
```
Hi Dylan, Great PR.. Just a small suggestion: in the type check for `invert_spike_types`, instead of raising a TypeError only when it's a `dict`, it would be clearer to ensure it's either a `string` or a `list`